Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageConfirmEditButton.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// Button used for confirming editing messages in ComposerView.
+public typealias ChatMessageConfirmEditButton = _ChatMessageConfirmEditButton<NoExtraData>
+
+/// Button used for confirming editing messages in ComposerView.
+open class _ChatMessageConfirmEditButton<ExtraData: ExtraDataTypes>: _Button, UIConfigProvider {
+    override open func setUpLayout() {
+        let normalStateImage = uiConfig.images.messageComposerSendEditedMessage
+        setImage(normalStateImage, for: .normal)
+        
+        let buttonColor: UIColor = uiConfig.colorPalette.inactiveTint
+        let disabledStateImage = uiConfig.images.messageComposerSendEditedMessage.tinted(with: buttonColor)
+        setImage(disabledStateImage, for: .disabled)
+    }
```

Comment:
Maybe this should belong in the `defaultAppearance` life cycle? 🤔 

Another thought: I think we probably could get rid of these `appearance` lifecycles and have only one lifecycle to set up all the UI. The way it is right now causes confusion and redundancy since you can do it in multiple ways. In some scenarios is also harder to customize, especially if you specify a custom `UIButton` in the `UIConfig` and then our views override the border color anyways. Just throwing here some thoughts 😄